packages/zoe/test/swingsetTests/upgradeCoveredCall/vat-ertp-service.js

```diff
@@ -1,9 +1,12 @@
 import { Far } from '@endo/marshal';
+import { AmountMath } from '@agoric/ertp';
 import { vivifyErtpService } from '@agoric/ertp/test/swingsetTests/ertpService/vat-ertp-service.js';
 
 export const buildRootObject = async (vatPowers, _vatParams, baggage) => {
+  const exportableAmountMath = Far('AmountMath', { ...AmountMath });
```

Comment:
Why do we need an `exportableAmountMath`?  AmountMath was designed to be usable locally in every vat and produce copy objects that can be sent around seamlessly.